server.js

```diff
@@ -10,7 +12,7 @@ const serveCompressedStaticAssets = require('./server/static-assets.mv.js');
 const sanitizeFilename = require('sanitize-filename');
 const asyncHandler = require('express-async-handler');
 
-const brewAccessTypes = ['edit', 'share', 'raw'];
+const brewAccessTypes = ['edit', 'share', 'raw', 'view'];
```

Comment:
What is the `view` type meant to do? I don't see `raw` being used anywhere now.

Reply:
I effectively replaced the use of `raw` with `view` since the metadata block would need to be removed and re-added to handle source/download. Raw doesn't really have a use anymore that I can think of so if it makes sense I'm happy to remove it.

Reply:
Yeah if we aren't using it go ahead and remove it.